politeiawww/cmd/politeiaverify/politeiaverify.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/decred/dcrtime/merkle"
+	"github.com/decred/politeia/util"
+)
+
+type record struct {
```

Comment:
Shouldn't these use the types defined in the pi API instead of redefining them?

Reply:
I mistakenly thought that we needed to keep this tool api-agnostic as it was initially, without any knowledge of the types defined there, but I guess there is no reason to do this indeed.

Reply:
So turns out we do need to define custom proposal/comment types. In it we define only the fields we do use on the verification, and besides that, the `serverpublickey` included in the bundle, and that is not included in the `pi` api definitions. This allows us to unmarshal the data correctly

Reply:
Ok can you add comments to the structs explaining why they don't use the pi API structs?